nature/forms.py

```diff
@@ -3,6 +3,21 @@
 
 
 class FeatureForm(forms.ModelForm):
+    name = forms.CharField(
+        widget=forms.Textarea,
+        max_length=80,
+        required=False,
+    )
+    description = forms.CharField(
+        widget=forms.Textarea,
+        max_length=255,
+        required=False,
+    )
+    notes = forms.CharField(
```

Comment:
Redefine the fields means the duplication of model field information (required, max_length), I think it is simpler to specify the widgets for the fields in Meta class if that's the only thing changed.